llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -1223,11 +1223,13 @@ void DwarfDebug::beginModule(Module *M) {
     }
 
     DenseSet<DIGlobalVariable *> Processed;
+//    CU.setGlobalVarMap(&GVMap);
```

Comment:
Why is this commented out, along with other large sections of code below?

Reply:
Consider using `#if 0` instead of `//` to disable large sections of code.

Reply:
Fixed